<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProxyChannelDlg</class>
 <widget class="QWidget" name="ProxyChannelDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProxyChannelDlg</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>本地端口:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>远端地址:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>远端端口:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLocalPort">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1090</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRemoteAddr">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>thinkingl.vicp.net</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRemotePort">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1090</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStart">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
